.github/workflows/check-runtimes-template-spec-chain.yml

```diff
@@ -0,0 +1,135 @@
+name: check-runtimes-template-spec-chain.yml
```

Comment:
I think this should be named to suggest it is related to minimal template, as you did for parachain template.

Reply:
I think we can create a custom workflow that receives parameters (mainly the package name and the associated runtime path), so that we don't have duplicate workflows for parachain/minimal templates. WDYT?

Reply:
I definitely agree with you on that. I first wanted to see the workflow pass 😅. To much cc cv